controllers/__init__.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+import os
+import re
+from typing import Dict, Optional
+import pymongo
+from pymongo.collection import ReturnDocument
+from pymongo.errors import AutoReconnect
+from tenacity import retry_if_exception_type, stop_after_attempt, wait_random
+from kytos.core import log
+from kytos.core.db import Mongo
+from kytos.core.retry import before_sleep, for_all_methods, retries
+from napps.kytos.mef_eline.db.models import EVCBaseDoc
+
+
+@for_all_methods(
+    retries,
+    stop=stop_after_attempt(
+        int(os.environ.get("MONGO_AUTO_RETRY_STOP_AFTER_ATTEMPT", "3"))
+    ),
+    wait=wait_random(
+        min=int(os.environ.get("MONGO_AUTO_RETRY_WAIT_RANDOM_MIN", "1")),
+        max=int(os.environ.get("MONGO_AUTO_RETRY_WAIT_RANDOM_MAX", "1")),
+    ),
+    before_sleep=before_sleep,
+    retry=retry_if_exception_type((AutoReconnect,)),
+)
+class ELineController:
+    """E-Line Controller"""
+
+    def __init__(self, get_mongo=lambda: Mongo()) -> None:
+        self.mongo = get_mongo()
+        self.db_client = self.mongo.client
+        self.db = self.db_client[self.mongo.db_name]
+    
+    def bootstrap_indexes(self) -> None:
+        """Bootstrap mef_eline relaeted indexes."""
+        index_tuples = [
+            ("evcs", [("circuit_scheduler", pymongo.ASCENDING)]),
```

Comment:
If the goal is to optimize for indexing individual insertions or reads by the id (in the future, since at it is it's upserting the entire doc), then it should be `circuit_scheduler.id`, could you double check?